### PR TITLE
feat: log to a file

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

File: cmd/move2kube/move2kube.go [hunk 1]
```diff
@@ -46,11 +48,20 @@ For more documentation and support, visit https://move2kube.konveyor.io/
 				logl = logrus.InfoLevel
 			}
 			logrus.SetLevel(logl)
+			if logFile != "" {
+				f, err := os.OpenFile(logFile, os.O_WRONLY|os.O_CREATE, common.DefaultFilePermission)
```
Comment (ashokponkumar) [inline cmd/move2kube/move2kube.go:52]: shouldn't it be opened in append mode?

File: cmd/move2kube/move2kube.go [hunk 2]
```diff
@@ -46,11 +48,20 @@ For more documentation and support, visit https://move2kube.konveyor.io/
...
+				if err != nil {
+					logrus.Fatalf("failed to open the log file at path %s . Error: %q", logFile, err)
+				}
+				logrus.SetOutput(io.MultiWriter(f, os.Stdout))
```
Comment (ashokponkumar) [inline cmd/move2kube/move2kube.go:56]: Check on if it uses stdout and stderr properly

**Review by ashokponkumar — CHANGES_REQUESTED** [review]
Few suggestions..

File: cmd/move2kube/move2kube.go (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply cmd/move2kube/move2kube.go]: done

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**